recipes-extended/spi-minnowmax-board/spi-minnowmax-board.bb

```diff
@@ -13,6 +13,37 @@ SRC_URI = "file://spi-minnow-cc2520.c \
            file://spi-minnow-board.c \
            file://spi-minnow-board.h \
            file://Makefile \
+           file://modules-load.d/cc2520.conf.sample \
+           file://modprobe.d/spi-minnow-cc2520.conf.sample \
+           file://modules-load.d/at86rf230.conf.sample \
+           file://modprobe.d/spi-minnow-at86rf230.conf.sample \
+           file://ostro-6lowpan.service \
+           file://ostro-6lowpan-start.sh \
+           file://ostro-6lowpan-stop.sh \
            "
 
+FILES_${PN} += " /usr/lib/modules-load.d/cc2520.conf.sample \
+                 /usr/lib/modules-load.d/at86rf230.conf.sample \
+                 /lib/modprobe.d/spi-minnow-cc2520.conf.sample \
+                 /lib/modprobe.d/spi-minnow-at86rf230.conf.sample \
+                 ${systemd_unitdir}/system/ostro-6lowpan.service \
+                 ${sysconfdir}/ostro-6lowpan/ostro-6lowpan-start.sh \
+                 ${sysconfdir}/ostro-6lowpan/ostro-6lowpan-stop.sh \
+               "
+
+# Sample configuring file
+do_install_append () {
+	install -d -m 755 ${D}${base_libdir}/modprobe.d
+	install -d -m 755 ${D}${libdir}/modules-load.d
+	install -m 0655 modules-load.d/cc2520.conf.sample ${D}${libdir}/modules-load.d/
+	install -m 0655 modprobe.d/spi-minnow-cc2520.conf.sample ${D}${base_libdir}/modprobe.d/
+	install -m 0655 modules-load.d/at86rf230.conf.sample ${D}${libdir}/modules-load.d/
+	install -m 0655 modprobe.d/spi-minnow-at86rf230.conf.sample ${D}${base_libdir}/modprobe.d/
+	install -d -m 755 ${D}${systemd_unitdir}/system/
+	install -m 0644 ostro-6lowpan.service ${D}${systemd_unitdir}/system/
+	install -d -m 755 ${D}${sysconfdir}/ostro-6lowpan/
+	install -m 0755 ostro-6lowpan-start.sh ${D}${sysconfdir}/ostro-6lowpan/
+	install -m 0755 ostro-6lowpan-stop.sh ${D}${sysconfdir}/ostro-6lowpan/
```

Comment:
As @mythi said, we rather not install it on etc. or anything since @pohly will work on the stateless. It can be in sbin if we expect people to manually call (I dislike) or libdir/ostro-scripts/ to hide it inside our own scope (I like it more)